src/Modules/UI/Dnn.EditBar.UI/editBar/resources/ContentEditorManager/Js/ModuleManager.js

```diff
@@ -73,6 +74,11 @@ if (typeof dnn === "undefined" || dnn === null) { dnn = {}; };
         _generateLayout: function () {
             return $('<a href="#" class="addModuleHandler" aria-label="Add Module"><span></span></a><a href="#" class="addExistingModuleHandler" aria-label="Add Existing Module"><span></span></a>');
         },
+
+        _generatePaneLabel: function() {
+            var paneName = this.$pane.attr('id').replace('dnn_', '');
+            return $('<span class="dnnPaneLabel" />').html(paneName);

```

Comment:
It appears strange to use the pane's id is used as HTML actually displayed to the users

Reply:
pane's id is the pane name.